cms/templates/js/certificate-details.underscore

```diff
@@ -27,6 +27,10 @@
          </header>
          <div class="signatory-details-list"></div>
     <% } %>
+    <div class="certificate-activation">
+        <button id="activate-certificate" class="activate-cert <% if(!is_active) { %> active-state <% } else { %>de-active-state <% } %>"><span><%= gettext("Activate") %></span></button>
```

Comment:
I think if is_active = True, then the first button should say "Active", and not be clickable. The second button should say "Deactivate" and be the only clickable button.  Conversely, if is_active = False, the first button should say "Activate" and be clickable.  The second button should say "Inactive" and not be clickable.